lib/special_route_publisher.rb

```diff
@@ -18,14 +18,27 @@ def publish(route_type, route)
     )
   end
 
+  def publish(route_type, route)
```

Comment:
Should this be `unpublish` @vanitabarrett ?

Reply:
@andrewgarner Yep, it should be. Also this is WIP, I haven't had a chance to test this out yet, not sure if it works...

Reply:
Ahh okay, no worries.

`GdsApi::PublishingApi::SpecialRoutePublisher` doesn't have `unpublish` as part of it's API so I think you might need to use https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api/publishing_api_v2.rb#L131-L162